src/AspNetCore/Src/forwardinghandler.cxx

```diff
@@ -1384,6 +1350,12 @@ FORWARDING_HANDLER::OnExecuteRequestHandler(
         pResponse->SetStatus(400, "Bad Request", 0, hr);
         goto Finished;
     }
+    else if (fInternalError)
+    {
+        // set a  special sub error code indicating loading application error
```

Comment:
This error should be only for the in process scenario. We can make a more specific log message regarding hosting the inprocess application?

Reply:
the sub error code should sufficient enough. We should not log per request based error to windows event log as it is too expensive. Ideally we should use etw which was prototyped